aimmo-game/tests/test_simulation/test_pickups.py

```diff
@@ -44,25 +47,85 @@ def test_effect_added(self):
 class TestHealthPickup(_BaseCases.BasePickupTestCase):
     pickup_class = pickups.HealthPickup
 
+    def test_invulnerability_is_applied(self):
+        self.apply_pickup()
+        self.assertIsInstance(self.pickup, pickups.HealthPickup)
+
     def test_health_increases(self):
         self.apply_pickup()
         self.assertEqual(self.avatar.health, 8)
 
     def test_serialise(self):
         self.assertEqual(self.pickup.serialise(), {'type': 'health', 'health_restored': 3})
 
+    def test_given_custom_health_pickup_increase_health_increases(self):
+        # Test 1 - increase of 5
+        self.assertEqual(self.avatar.health, 5)
```

Comment:
consider the following style of test:

// given
self.avatar.health = 10
self.set_custom_pickup(5)

// when
self.apply_pickup()

// then
self.assertEqual(self.avatar.health, 15)

You don't have to put the comments but the separation makes it clear what is happening.

Reply:
ok cool